modules/microshift-configuring-ovn.adoc

```diff
@@ -8,6 +8,9 @@
 
 An OVN-Kubernetes config file can be written to `/etc/microshift/ovn.yaml`. {product-title} uses default OVN-Kubernetes configuration values if an OVN-Kubernetes config file is not customized.
 
+//Q: does the file populate with values if created empty, or is none of this true?
```

Comment:
I do not see this file on my system, so I don't think anything creates it if the user does not provide one. Generally we would use built-in defaults, but not write them to the filesystem, and I think that's the case here.